accessibility_scanning/README.md

```diff
@@ -13,34 +17,54 @@ general:
 
 machine:
   environment:
-    ACCESSLINT_API_TOKEN: <API token from https://accesslint.com>
-    ACCESSLINT_GITHUB_USER: <GitHub user authenticated at https://accesslint.com>
     ACCESSLINT_MASTER_BRANCH: <Branch that you want accesslint to test against>
   node:
     version: 6.1.0
 
 dependencies:
+  pre:
+    - gem install bundler
   override:
     - npm install -g accesslint-cli
     - gem install accesslint-ci
 
 test:
   post:
-    - bundle exec jekyll serve --detach && bundle exec accesslint-ci scan http://localhost:4000
-
+    - bundle exec jekyll serve --detach
+    - bundle exec accesslint-ci scan http://localhost:4000
 ```
 
-Both the `ACCESSLINT_API_TOKEN` and `ACCESSLINT_GITHUB_USER` should be configured in the CircleCI settings of your repo and not added directly to the `circle.yml` file itself.
+The `ACCESSLINT_MASTER_BRANCH` should be set to the branch that PRs are being made to. For 18F pages projects, this will generally be `18f-pages`.
```

Comment:
@gemfarmer Didn't Jez just say 18F pages are getting deprecated this week? Should we use a different example like "master"?

Should it be this instead?
>The `ACCESSLINT_MASTER_BRANCH` should be set to the repo's default or "base" branch which all pull requests and commits are automatically made to.